projects/components/src/table/table.component.ts

```diff
@@ -453,12 +471,15 @@ export class TableComponent
     }
   }
 
-  private initializeUrl(): void {
-    if (this.paginator) {
-      this.onPageChange({
```

Comment:
How is this being done now?

Reply:
It didn't feel valuable to put the default page info in the url on load. I can buy that argument for something like the selected tab, but for page size I think the simpler url for the more common use case is better.

Reply:
Okay I agree with that.